### PR TITLE
Applying proguard in an android application.

### PR DESCRIPTION
**Pull request by diwakarsinghdiwakar**

### CODE REVIEW

File: source/_posts/Applying-Proguard-in-an-android-application.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Applying Proguard in an android application
```
Comment (Rajeev111974) [inline source/_posts/Applying-Proguard-in-an-android-application.md:2]: Title   tag character should be 50-60 character

File: source/_posts/Applying-Proguard-in-an-android-application.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+date: 2020-07-27 18:40:48
+author: Diwakar Singh
+category: Android
```
Comment (Rajeev111974) [inline source/_posts/Applying-Proguard-in-an-android-application.md:5]: Keyword should be linked with website nayan

File: source/_posts/Applying-Proguard-in-an-android-application.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+tags:
```
Comment (Rajeev111974) [inline source/_posts/Applying-Proguard-in-an-android-application.md:6]: Content   should have link to previous blog

File: source/_posts/Applying-Proguard-in-an-android-application.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+---
```
Comment (Rajeev111974) [inline source/_posts/Applying-Proguard-in-an-android-application.md:1]: Image should be optimized with Alt text

Comment (Rajeev111974) [inline source/_posts/Applying-Proguard-in-an-android-application.md:1]: Add a Featured Image

**Comment by Rajeev111974** [comment]
HI diwakar,

 

Please see the blog and check the comment. We have marked some changes there which needs to complete.

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

File: source/_posts/Applying-Proguard-in-an-android-application.md (hunk 1 above)
Comment (diwakarsinghdiwakar) [reply source/_posts/Applying-Proguard-in-an-android-application.md:2]: Updated

File: source/_posts/Applying-Proguard-in-an-android-application.md (hunk 3 above)
Comment (diwakarsinghdiwakar) [reply source/_posts/Applying-Proguard-in-an-android-application.md:6]: Added

File: source/_posts/Applying-Proguard-in-an-android-application.md (hunk 4 above)
Comment (diwakarsinghdiwakar) [reply source/_posts/Applying-Proguard-in-an-android-application.md:1]: done

Comment (diwakarsinghdiwakar) [reply source/_posts/Applying-Proguard-in-an-android-application.md:1]: done

**Pull request merged**